app/test_main.py

```diff
@@ -1,3 +1,84 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,expected",
+        [
+            (0, 0, [0, 0]),
+            (14, 14, [0, 0]),
+            (15, 15, [1, 1]),
+            (23, 23, [1, 1]),
+            (24, 24, [2, 2]),
+            (27, 27, [2, 2]),
+            (28, 28, [3, 2]),
+            (100, 100, [21, 17])
+        ],
+        ids=[
+            "Age of the cat/dog == 0/0 should convert into 0.",
+            "Age of the cat/dog == 14/14 should convert into 0.",
+            "Age of the cat/dog == 15/15 should convert into 1.",
+            "Age of the cat/dog == 23/23 should convert into 1.",
+            "Age of the cat/dog == 24/24 should convert into 2.",
+            "Age of the cat/dog == 27/27 should convert into 2.",
+            "Age of the cat/dog == 28/28 should convert into 3/2.",
+            "Age of the cat/dog == 100/100 should convert into 21/17."
+        ]
+    )
+    def test_get_human_age(self,
+                           cat_age: int,
+                           dog_age: int,
+                           expected: list[int]) -> None:
+        assert get_human_age(cat_age, dog_age) == expected, (
+            f"Age of cat {cat_age} and age of dog "
+            f"{dog_age} should return {expected}"
+        )
+
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,expected",
+        [
+            (-2, 5, [0, 0]),
+            (10, -1, [0, 0]),
+            (-15, -6, [0, 0]),
+            (550, 550, [133, 107]),
+            (1200, 1500, [296, 297])
```

Comment:
The expected results for negative and very high values should be carefully considered. The current expected values may not align with the realistic or intended behavior of the `get_human_age` function. Ensure that the function's logic is correctly handling these edge cases.